packages/exos-scripts/src/scripts/lint/index.ts

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 import path from "path";
-import { CLIEngine } from "eslint";
+const { ESLint } = require("eslint");
 import { SOURCE_PATH } from "../../common/paths";
 import getConfigToUse from "../../common/getConfigToUse";
 import eslintrcReact = require("./.eslintrc.react");
```

Comment:
```suggestion
import { SOURCE_PATH } from "../../common/paths";
import getConfigToUse from "../../common/getConfigToUse";
import eslintrcReact = require("./.eslintrc.react");

// Use require because the ESLint types aren't updated yet.
const { ESLint } = require("eslint");
```